Adaptors/Amqp/src/QueueMessageHandler.cs

```diff
@@ -59,7 +59,7 @@ public class QueueMessageHandler : IQueueMessageHandler
   }
 
   /// <inheritdoc />
-  public CancellationToken CancellationToken { get; }
+  public CancellationToken CancellationToken { get; set; }
```

Comment:
Why set ?

Reply:
I don't remember the usage of the set part.

Reply:
I put the set to pass the right cancellation token in the RabbitMQ adapter, its only used there.